src/client/components/features/CampagneApprentissage/RaisonsDeChoisirApprentissage/RaisonsDeChoisirApprentissage.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Container } from 'src/client/components/layouts/Container/Container';
+import { Tuile } from 'src/client/components/ui/Tuile/Tuile';
+
+import styles from '~/client/components/features/CampagneApprentissage/RaisonsDeChoisirApprentissage/RaisonsDeChoisirApprentissage.module.scss';
+import { IconName } from '~/client/components/ui/Icon/Icon';
+
+interface RaisonsDeChoisirApprentissageProps {
+	raisons: {
+		iconName: IconName;
+		text: string;
+	}[];
```

Comment:
```suggestion
	raisons: Raison[];
```

du coup tu pourras utiliser le type Raison dans les deux composants